### ISSUE TITLE
editing / customizing the footer....

### ISSUE THREAD

**Issue by 4evermaat** [giansi/gravstrap-theme#17]
I wanted to know how to edit the footer to make it look more like [this page here](http://www.DirectFx.com/en) in this screenshot:
![directfx bottom navbar and site map](https://cloud.githubusercontent.com/assets/830306/13969518/f7a384d4-f0b5-11e5-870e-53d58bf304fb.gif)

Notice how for the text links at the bottom of the page, they list each page individually in columns.  Is that something that can be automated in gravstrap?  Or do I have to update all the links manually when I add/delete a page on my website?  It seems like I would implement a page module for this or some sort of reusable gravstrap section.

I looked in the `/pages/common/common.md` and I see some footer code there `[g-footer-one name="footer" render=false]`,

![gravstrap theme edit footer](https://cloud.githubusercontent.com/assets/830306/13970404/1e4f5e0a-f0bb-11e5-9888-7efac812a4bf.gif)

but how does gravstrap know to parse some of the navigation links in the footer?  I do not see which arguments control the navigation links in footer?

**Comment by giansi** [comment]
You must add a new twig template to your theme to handle your new footer or, better, add a new shortcode to handle it. If you choose the second option, please consider to add it to Gravstrap and share with others. :-)

**Comment by 4evermaat** [comment]
could you give me an example of each? (twig template and custom shortcode)?  I couldn't find the twig template for gravstrap footer that deals with rendering the footer.  I see the `common.md` in the `user/pages/common` folder.

But the way you wrote your footer, you didn't allow someone to edit it (I cannot remove the breadcrumbs at the bottom.  I don't know if this was intentional.

I thought about writing a module component that could be used as a footer.  But if the shortcode method is easier, and I can just drop it into any page I need to.....I'll do it.

Sorry if I'm such a bother.  I'm not really a programmer.  I can understand some code, but i can work from template and when I know which parameters I can change, how they affect the page, etc.  I would have worked with the existing footer if I could edit the forced navigation.  See screenshot:
![gravestrap footer edit footer navigation](https://cloud.githubusercontent.com/assets/830306/14069643/d6353120-f4c6-11e5-9fc0-c8580d8c6ab4.gif)
where do I go to edit that?  I do not see any obvious switch in the common.md file.

**Comment by 4evermaat** [comment]
I see it now in the `/plugins/gravstrap/templates/footer/footer_one.html.twig` and `/plugins/gravstrap/templates/modules/partials/footer_navigation.html.twig`   .  Took me over 10 days to figure it out (I kept looking in the `themes` folder)!!!  Water under the bridge.

I couldn't figure out what was rendering the footer and then I inspected the element very carefully in Chrome browser and it showed me the plugin that had the css.  I found it.
